core/src/main/java/bisq/core/trade/TradeUtil.java

```diff
@@ -164,14 +164,13 @@ public String getMarketDescription(Trade trade) {
     }
 
     public String getPaymentMethodNameWithCountryCode(Trade trade) {
-        String paymentMethodDescription = "";
-        if (trade != null) {
-            Offer offer = trade.getOffer();
-            checkNotNull(offer);
-            checkNotNull(offer.getPaymentMethod());
-            paymentMethodDescription = offer.getPaymentMethodNameWithCountryCode();
-        }
-        return paymentMethodDescription;
+        if (trade == null)
+            return "";
+
+        Offer offer = trade.getOffer();
```

Comment:
This is not required, but I do like the notation
`Offer offer = checkNotNull(trade.getOffer());`